test/test_xla_sharding.py

```diff
@@ -1,26 +1,24 @@
 import os
 import sys
 
+import unittest
+import numpy as np
+
 import torch
 import torch_xla
 import torch_xla.core.xla_model as xm
 import torch_xla.utils.utils as xu
 import torch_xla.experimental.xla_sharding as xs
 from torch_xla.experimental.xla_sharded_tensor import XLAShardedTensor
-import unittest
-import numpy as np
-
-# A placeholder for future sharding tests
 
 
 class XlaShardingTest(unittest.TestCase):
 
```

Comment:
can we add a assert on `PJRT_DEVICE` somewhere in this file, we won't support SPMD on xrt.

Reply:
+1, though I prefer skip_test over assertion failure. I will add this to the testing conditions.

Reply:
Done